Example/Tests/BoolOperationsTests.swift

```diff
@@ -0,0 +1,156 @@
+//
+//  BoolOperationsTests.swift
+//  MobilliumDateFormatter_Tests
+//
+//  Created by Mehmet Salih Aslan on 17.03.2021.
+//  Copyright © 2021 CocoaPods. All rights reserved.
```

Comment:
We can change copyright to Mobillium
```suggestion
//  Copyright © 2021 Mobillium. All rights reserved.
```